src/index.js

```diff
@@ -0,0 +1,17 @@
+import _, { remove } from 'lodash';
+// import printMe from './print.js';
```

Comment:
- [ ] please try to  remove this line of unused code to keep to file clean and professional